File: hello.py
class Greeter:
    def __init__(self, name):
        self.name = name

    def greet(self):
        print(f"Hello, World from Python OOP! (Greetings to {self.name})")

if __name__ == "__main__":
    greeter = Greeter("GitHub Automation")
    greeter.greet()

File: hello.php
<?php

class Greeter {
    private $name;

    public function __construct($name) {
        this->name = $name;
    }

    public function greet() {
        echo "Hello, World from PHP OOP! (Greetings to " . $this->name . ")\n";
    }
}

$greeter = new Greeter("GitHub Automation");
$greeter->greet();
?>
